src/doublyLinkedList.js

```diff
@@ -0,0 +1,136 @@
+import Node from './node'
+
+export default class DoublyLinkedList {
+  constructor() {
+    this.length = 0
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode = () => {
+    return this.head
+  }
+
+  getTailNode = () => {
+    return this.tail
+  }
+
+  contains = (value) => {
+    if (this.isEmpty()) return null
```

Comment:
if statements should have explicit declaration with curly braces for consistency, and in the case you had to refactor, it will be easier